rust/operator-binary/src/controller.rs

```diff
@@ -20,12 +20,15 @@ use product_config::{
 use snafu::{OptionExt, ResultExt, Snafu};
 use stackable_operator::builder::resources::ResourceRequirementsBuilder;
 use stackable_operator::builder::{
-    PodSecurityContextBuilder, SecretOperatorVolumeSourceBuilder, VolumeBuilder,
+    ObjectMetaBuilderError, PodSecurityContextBuilder, SecretOperatorVolumeSourceBuilder,
+    SecretOperatorVolumeSourceBuilderError, VolumeBuilder,
 };
 use stackable_operator::client::GetApi;
 use stackable_operator::commons::authentication::tls::{CaCert, TlsVerification};
 use stackable_operator::commons::s3::S3ConnectionSpec;
+use stackable_operator::commons::secret_class::SecretClassVolumeError;
 use stackable_operator::k8s_openapi::api::core::v1::SecretVolumeSource;
+use stackable_operator::kvp::{LabelError, Labels};
```

Comment:
Can we summarize these imports?

Reply:
Do you mean like with `::*`?

I guess that raises the debate of explicit vs implicit imports.

Perhaps for the stuff used in every operator, we can make a prelude module in operator-rs which re-exports stuff.

Reply:
Use a grouped import is I guess what he meant. So like `use foo::{a, b, c d::{e, f}}`.

Reply:
yes @Techassi

Reply:
Can do :+1:

Reply:
Resolved in dca6a44ffdf8951892e9921a5aeb5b97829bb1b9